### PR TITLE
intoduce `substrate-frontier-evm` package

### PR DESCRIPTION
**Pull request by belopash**

### CODE REVIEW

**Review by dzhelezov — APPROVED** [review]
LGTM modulo error messages

File: util/substrate-frontier-evm/src/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import assert from 'assert'
+import {Call, ChainContext, Event} from './interfaces'
```
Comment (tmcgroul) [inline util/substrate-frontier-evm/src/transaction.ts:2]: `Event` is unused

File: util/substrate-frontier-evm/src/log.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import {assert} from 'console'
```
Comment (tmcgroul) [inline util/substrate-frontier-evm/src/log.ts:1]: Shouldn't it be `import assert from 'assert'`?

**Review by tmcgroul — CHANGES_REQUESTED** [review]

**Comment by tmcgroul** [comment]
@belopash Also please add a README

**Pull request merged**